logfire/_internal/config.py

```diff
@@ -207,12 +208,18 @@ class CodeSource:
     revision: str
     """The git revision of the code e.g. branch name, commit hash, tag name etc."""
 
-    root_path: str
+    root_path: str = field(default='.')
```

Comment:
In meta I see this resource attribute isn't present. I think the default value should be `''` and the resource attribute should only be included if it's nonempty.

Also, is a `field` needed?

```suggestion
    root_path: str = ''
```